content/en/docs/how-tos/external-images.md

```diff
@@ -16,8 +16,12 @@ If the source image is open to the public, we can mirror the image by adding it
 gcr.io/k8s-staging-boskos/boskos:latest registry.ci.openshift.org/ci/boskos:latest
 {{< / highlight >}}
 
-The hourly periodic job `periodic-image-mirroring-supplemental-ci-images` mirrors all the images defined in the mapping files.
+The hourly periodic job `periodic-image-mirroring-supplemental-ci-images`
```

Comment:
Could we linkify it to https://prow.ci.openshift.org/?job=periodic-image-mirroring-supplemental-ci-images

Reply:
Linkified.